horovod/keras/callbacks.py

```diff
@@ -99,9 +99,14 @@ def __init__(self, multiplier, start_epoch=0, end_epoch=None, staircase=True,
             steps_per_epoch: The callback will attempt to autodetect number of batches per
                              epoch with Keras >= 2.0.0. Provide this value if you have an older
                              version of Keras.
+            initial_lr: Initial learning rate at the start of training.
```

Comment:
In the doc above for this callback, could we explicitly mention that when using a constant multiplier, it applies to the `initial_lr` and not the current learning rate? (because different behavior than TF Keras 2.2 callback which takes current lr)

Reply:
same for TF Keras file

Reply:
Nevermind, I missed the description for class at https://github.com/horovod/horovod/blob/2a3f43f35c840d7e8cfa9674a051ffa53be9918d/horovod/tensorflow/keras/callbacks.py#L75